testsuite/python/h5md.py

```diff
@@ -92,157 +127,99 @@ def test_metadata(self):
+            [x for (_, x) in sorted(zip(self.py_id, self.py_charge))], charges)
 
     def test_vel(self):
         """Test if velocities have been written properly."""
         np.testing.assert_allclose(
-            np.array([[1.0, 2.0, 3.0] for _ in range(npart)]),
+            np.array([[1.0, 2.0, 3.0] for _ in range(N_PART)]),
             np.array([x for (_, x) in sorted(zip(self.py_id, self.py_vel))]),
             err_msg="Velocities not written correctly by H5md!")
 
     @utx.skipIfMissingFeatures(['EXTERNAL_FORCES'])
     def test_f(self):
         """Test if forces have been written properly."""
         np.testing.assert_allclose(
-            np.array([[0.1, 0.2, 0.3] for _ in range(npart)]),
+            np.array([[0.1, 0.2, 0.3] for _ in range(N_PART)]),
             np.array([x for (_, x) in sorted(zip(self.py_id, self.py_f))]),
             err_msg="Forces not written correctly by H5md!")
 
     def test_bonds(self):
         """Test if bonds have been written properly."""
-        self.assertEqual(len(self.py_bonds), npart - 1)
+        self.assertEqual(len(self.py_bonds), N_PART - 1)
 
-        for i in range(npart - 1):
+        for i in range(N_PART - 1):
             bond = [x for x in self.py_bonds if x[0] == i][0]
             self.assertEqual(bond[0], i + 0)
             self.assertEqual(bond[1], i + 1)
 
+    def test_script(self):
+        with open(sys.argv[0], 'r') as f:
+            ref = f.read()
+        data = self.py_file['parameters/files'].attrs['script'].decode('utf-8')
+        self.assertEqual(data, ref)
+
     def test_units(self):
         self.assertEqual(
             self.py_file['particles/atoms/id/time'].attrs['unit'], b'ps')
         self.assertEqual(
             self.py_file['particles/atoms/position/value'].attrs['unit'], b'm')
-        if espressomd.has_features(['ELECTROSTATICS']):
-            self.assertEqual(
-                self.py_file['particles/atoms/charge/value'].attrs['unit'], b'e')
-        if espressomd.has_features(['MASS']):
-            self.assertEqual(
-                self.py_file['particles/atoms/mass/value'].attrs['unit'], b'u')
-        self.assertEqual(
-            self.py_file['particles/atoms/force/value'].attrs['unit'],
-            b'm u ps-2')
-        self.assertEqual(
-            self.py_file['particles/atoms/velocity/value'].attrs['unit'],
+        if espressomd.has_features(['ELECTROSTATICS']):	
+            self.assertEqual(	
+                self.py_file['particles/atoms/charge/value'].attrs['unit'], b'e')	
+        if espressomd.has_features(['MASS']):	
+            self.assertEqual(	
+                self.py_file['particles/atoms/mass/value'].attrs['unit'], b'u')	
+        self.assertEqual(	
+            self.py_file['particles/atoms/force/value'].attrs['unit'],	
+            b'm u ps-2')	
+        self.assertEqual(	
+            self.py_file['particles/atoms/velocity/value'].attrs['unit'],	
```

Comment:
trailing whitespaces

Reply:
that comes from automatic formatting. I don't know why this happened.